OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -122,32 +123,97 @@ public FileNotFoundSequence(FileNotFoundException exception)
 		float ISpriteSequence.GetAlpha(int frame) { throw exception; }
 	}
 
+	[Desc("This is a default sprite sequence.")]
 	public class DefaultSpriteSequence : ISpriteSequence
 	{
 		static readonly WDist DefaultShadowSpriteZOffset = new WDist(-5);
 		protected Sprite[] sprites;
 		readonly bool reverseFacings, transpose;
 		readonly string sequence;
-		readonly float[] alpha;
 
 		protected readonly ISpriteSequenceLoader Loader;
 
 		public string Name { get; private set; }
+
+		[Desc("")]
 		public int Start { get; private set; }
-		public int Length { get; private set; }
+
+		[Desc("")]
+		public int Length { get; private set; } = 1;
+
+		[Desc("")]
 		public int Stride { get; private set; }
-		public int Facings { get; private set; }
-		public int Tick { get; private set; }
+
+		[Desc("")]
+		public int Facings { get; private set; } = 1;
+
+		[Desc("")]
+		public int Tick { get; private set; } = 40;
+
+		[Desc("")]
```

Comment:
```suggestion
		[Desc("Value controlling the Z-order. A higher values means rendering ontop of other sprites at the same position. Use power of 2 values to avoid glitches.")]
```